cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxActivity.java

```diff
@@ -23,6 +23,8 @@ of this software and associated documentation files (the "Software"), to deal
  ****************************************************************************/
 package org.cocos2dx.lib;
 
+import java.io.ByteArrayOutputStream;
+
```

Comment:
I think it is better to add a separate java file for it, or Cocos2dxActivity.java will be complex.